### ISSUE TITLE
Website doesn't work...

### ISSUE THREAD

**Issue by maeffew** [xDae/styled-bootstrap#105]
There doesn't seem to be a server named http://styled-bootstrap.js.org

**Comment by xDae** [comment]
true, at the moment I dont have time to spend in this project, but i would like to continue with it, i dont spect to abandon it :)

**Comment by kopax** [comment]
Hi @xDae! Are you willing to help and implement your good ideas within a project with more contributors ? It's quite heavy for 1 person and you had some good api implementation idea we could recycle. I hope you have solved your job change issue. https://github.com/bootstrap-styled/bootstrap-styled.github.io
